wiki/Website-Media.md

```diff
@@ -41,3 +43,44 @@ Regarding the format and settings of the "Export" panel in Figma, please follow
   - is it an image that resizes when the viewport changes? speak with the developer and decide what is the right size to use
 
 If you have any doubts or problems exporting the files, speak with one of the designers of the HDS team.
+
+## Optimizing images
+
+Prior to adding new images to the website the assets should be optimized to reduce their file size and increase our web performance. Figma's export feature doesn't allow for much fine-grained control regarding image quality and optimizing for speed on the web. [ImageOptim](https://imageoptim.com) works great, is free, easy to use, and with the correct settings can result in a more than _50% reduction_ in file size per image.
+
+1. Install the desktop app for your specific operating system.
+2. Open the application settings; generally most of the default settings work out of the box, but for reference:
+    1. In the quality tab, enable `Lossy minification`
+    2. Set the image qualities for .jpg, .png, and .gif to 85-90%. This results in a high quality, high resolution image (using the guidelines outlined above), while removing unnecessary metadata and compressing the filesize to something more managable for the web.
+3. Within the app, either add images to the queue with the plus icon in the lower left corner, or simply drag and drop files from your file browser into the window.
```

Comment:
```suggestion
3. Within the app, add images either using the plus icon in the lower left corner or by dragging and dropping files from your file browser into the window.
```

Reply:
I'm thinking THAT this part is probably understood so could be deleted: "Within the app"